images/image-set-type.html

```diff
@@ -30,6 +33,9 @@
 
     <textarea class="playable playable-css" style="height: 150px;">
```

Comment:
Have you tested that this does not cause a scrollbar? I think height: 150px will not be tall enough for people to see the code.